cnv_somatic_pair_workflow.b37.inputs.json

```diff
@@ -0,0 +1,121 @@
+{
```

Comment:
One general question: are these JSONs working off the 4.0.0.0 release?  We've already updated the WDL in the 4.0.1.0 release.  In general, what is the plan for keeping these JSONs in sync?

Reply:
The wdls was obtained from the 4.0.0.0 tag, but the wdl can be updated to the latest 4.0.1.0 tag and use the latest docker.

Reply:
now updated to 4.0.1.2